src/js/gi/store/index.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
We can use the common store for this instead of having a separate store for GIBCT